### PR TITLE
Add a composer.json.

### PR DESCRIPTION
**Pull request by casperh1nl**
If i want to install this plugin with composer we cannot. To fix that you can add a composer.json.

In the composer.json we set the type to prestashop-module so it will be picked up by https://github.com/composer/installers on instalation.

### CODE REVIEW

**Comment by mikededecker1** [comment]
Hi @casperh1nl 

Thank you for this new feature. We want to add this feature to our current develop branch.
Currrently we have found some issues with the name and the description of the composer file.
Therefore i added some code suggestions that will solve these issues.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "multi-safepay/presta-shop",
```
Comment (mikededecker1) [inline composer.json]: For a friendly composer require. I want to remove the dash (-)
```suggestion
  "name": "multisafepay/prestashop",
```

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "description": "Prestashop 1.7 compatible MultiSafepay module\n",
```
Comment (mikededecker1) [inline composer.json]: The description I want to follow the description of our Magento 2 composer file
```suggestion
  "description": "MultiSafepay module for PrestaShop 1.7",
```

**Review by mikededecker1 — CHANGES_REQUESTED** [review]

**Comment by casperh1nl** [comment]
Done!

**Review by mikededecker1 — APPROVED** [review]
Thank you for your changes. 👍

**Pull request merged**